spec/models/money_group_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe MoneyGroup, type: :model do
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement [previous reviewer's suggestion](https://github.com/Elijahdre/Budget-app/pull/2/files#r1117945033) and create `Integration` tests for your project 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/221375523-089cad42-1a79-4fab-bd40-3172a65c1d4e.png)